src/components/Form.js

```diff
@@ -0,0 +1,144 @@
+import Button from './Button';
+import styled from 'styled-components';
+import { useState } from 'react';
+
+export default function Form({ onSubmitIngredients }) {
+  const [ingredient1, setIngredient1] = useState('');
+  const [ingredient2, setIngredient2] = useState('');
+  const [ingredient3, setIngredient3] = useState('');
+  const [ingredient4, setIngredient4] = useState('');
+  const [ingredient5, setIngredient5] = useState('');
+  let arr = [];
+
+  function onSubmit(event) {
+    event.preventDefault();
+    if (
+      ingredient1 &&
+      ingredient2 &&
+      ingredient3 &&
+      ingredient4 &&
+      ingredient5
+    ) {
+      arr.push(ingredient1, ingredient2, ingredient3, ingredient4, ingredient5);
+    } else if (ingredient1 && ingredient2 && ingredient3 && ingredient4) {
+      arr.push(ingredient1, ingredient2, ingredient3, ingredient4);
+    } else if (ingredient1 && ingredient2 && ingredient3) {
+      arr.push(ingredient1, ingredient2, ingredient3);
+    } else if (ingredient1 && ingredient2) {
+      arr.push(ingredient1, ingredient2);
+    } else {
+      alert('You need to put in your first and second ingredient');
+      return;
+    }
+
+    onSubmitIngredients(arr);
+
+    setIngredient1('');
+    setIngredient2('');
+    setIngredient3('');
+    setIngredient4('');
+    setIngredient5('');
+  }
+
+  return (
+    <StyledForm onSubmit={onSubmit}>
```

Comment:
I think your Form needs something to make it accessible. Like aria-labelledby (maybe you could link a screen-reader-only span with a description of your form) or any other accessibility methods 🙂